plugins/out_stackdriver/stackdriver.h

```diff
@@ -48,6 +48,11 @@
 
 #define LOCAL_RESOURCE_ID_KEY "logging.googleapis.com/local_resource_id"
 
+#define NO_STREAM 0
+#define STREAM_STDOUT 1
+#define STREAM_STDERR 2
+#define STREAM_UNKNOWN 3
```

Comment:
What is the difference between `STREAM_UNKNOWN` and `NO_STREAM`? Should we combine the two?

Reply:
removed

Reply:
updated